guiEditor/src/components/propertyTab/propertyTabComponent.tsx

```diff
@@ -126,15 +127,18 @@ export class PropertyTabComponent extends React.Component<IPropertyTabComponentP
     }
 
     save() {
+        this.props.globalState.guiTexture.removeControl(this.props.globalState.workbench.artBoardBackground);

```

Comment:
A comment would be nice here and on 137 as I am not sure why we need this and the unshift ?